src/components/PixelLink.astro

```diff
@@ -6,6 +6,13 @@ const { variant = 'primary' } = Astro.props;
 const { before, after } = Astro.slots;
 ---
 
+<!-- Blocking script -->
+<script global hoist>
```

Comment:
Does `global` do anything?

Reply:
Great catch, it doesn't!  I was trying out a bunch of different script setups and didn't notice this had both flags :+1: